MATLAB/Utilities/addCTnoise.m

```diff
@@ -85,6 +85,14 @@
                 m=val(1);
                 sigma=val(2);
             end
+        
+        case 'Implementation'
+            if default
+                implementation='cuda';
+            else
+                implementation=val;
+            end
+            
         otherwise
             error('CBCT:addnoise:InvalidInput',['Invalid input name:', num2str(opt),'\n No such option in OS_SART_CBCT()']);

```

Comment:
`            error('CBCT:addnoise:InvalidInput',['Invalid input name:', num2str(opt),'\n No such option in addCTnoise()']);`